excercise9/itemmaster/Main.java

```diff
@@ -0,0 +1,58 @@
+package itemmaster;
+
+import java.sql.Connection;
+import java.util.List;
+import java.util.Scanner;
+import jdbcutility.JDBCUtility;
+
+public class Main {
+	public static void main(String[] args) {
+		String item_name;
+		float item_price;
+		int unit;
+		
+		Scanner scanner = new Scanner(System.in);
```

Comment:
Can you add error handling and input validation?

Reply:
added error handling and input validation